DOCS-8785: Reorg PANA docs (#29604)
DataDog/documentation: content/en/product_analytics/_index.md

By default, Product Analytics data is retained for 15 months. Learn more about [Privacy at Datadog][1].

## Navigating the Product Analytics UI

> Each Product Analytics feature provides context into your users journey. This section describe the context each feature can provide for your individual use case.

Reviewer: ```suggestion
Each Product Analytics feature provides context into your users' journeys. This section describes the context each feature can provide for your individual use case.
```

### Understand product adoption

> The [Home][3] page gives you a birds eye view of your users' activity and a pulse into your product's adoption. This is where you most often land when accessing Product Analytics.

Reviewer: ```suggestion
The [Home][3] page gives you a bird's-eye view of your users' activity and a pulse into your product's adoption. This is where you most often land when accessing Product Analytics.
```

{{< img src="/product_analytics/pana_home_page.png" alt="Understand end-to-end conversions with Funnel Analysis.">}}

By default, this page displays the `active users`, `page views` and `average time spent by user` charts, but you have the ability to add additional charts or a dashboard. You can navigate to anywhere in Product Analytics from the home page. 

### Track conversion rates and their evolution over time

> The Product analytics charts help you visualize your users' journey as they use your product.

Reviewer: ```suggestion
The Product Analytics charts help you visualize your users' journey as they use your product.
```

### Visualize the most and least interacted with buttons on a given page
Suppose you want to make changes to your application interface but want to first understand how users navigate in the page. Is there a specific path they take more than others? Can you make user actions and flows smoother? The following features can help you capture and replay your users' browsing experience to inform your product change decisions. 

{{< img src="/product_analytics/pana_session_replay_page.png" alt="Capture and replay your users' browsing experience to inform your product design decisions.">}}

[Session replay][11] 
: Expands your user experience monitoring by allowing you to capture and visually replay the web browsing or mobile app experience of your users. <br><br>This is beneficial for _error identification_, _reproduction_, and _resolution_, and provides insights into your application’s usage patterns and design pitfalls

[Heatmaps][7]

> : Also written as heat maps is a visualization of your user’s interactions overlaid on Session Replay data. Product Analytics has three different types of heatmaps: [Click maps][7], [Top elements][7], [Scroll maps][7]. <br><br> Use heatmaps to review complex data at a glance, gaining insights around optimizing your user experience.

Reviewer: It's probably OK not to mention "heatmaps" vs. "heat maps", but if you want to, slight edit:
```suggestion
: Also written as "heat maps," this is a visualization of your user’s interactions overlaid on Session Replay data. Product Analytics has three different types of heatmaps: [Click maps][7], [Top elements][7], [Scroll maps][7]. <br><br> Use heatmaps to review complex data at a glance, gaining insights around optimizing your user experience.
```
